android/java/org/chromium/chrome/browser/app/domain/PortfolioModel.java

```diff
@@ -79,9 +80,9 @@ public PortfolioModel(Context context, TxService txService, KeyringService keyri
         mBraveWalletService = braveWalletService;
         mAssetRatioService = assetRatioService;
         mSharedData = sharedData;
-        _mNftModels = new MutableLiveData<>(Collections.emptyList());
+        _mNftModels = new MutableLiveData<>();
```

Comment:
Can you please elaborate on the part where you state "than adding null checks", please?
Because I haven't added any null checks, this sentence left confused 🤔

Reply:
Thanks for asking. I had linked the [null checks](https://github.com/brave/brave-core/pull/17967#discussion_r1165352241) comment to add null check instead of adding further details.

Just realise that practically there's no issue with the current flow but if someone tries to use `getValue` directly then `_mNftModels` will be `null` now instead of empty list so good practice to use a default value when it make sense/possible. Changing this to nit, feel free to ignore.